### ISSUE TITLE
Cloudshell doesn't find PiHole anymore

### ISSUE THREAD

**Issue by oradke** [MichaIng/DietPi#5810]
Cloudshell relies on /run/pihole-FTL.port to get the current status. However, /run/pihole-FTL.port has been deprecated:

```
PORTFILE=/run/pihole-FTL.port[¶](https://docs.pi-hole.net/ftldns/configfile/#file_PORTFILE)
This option is deprecated as FTL does not write any port file anymore. Please parse pihole-FTL.conf if you need to check if a custom API port is set.
```
Source: https://docs.pi-hole.net/ftldns/configfile/

**Comment by Joulinar** [comment]
Many thanks for reporting it. We will have a look.

**Comment by MichaIng** [comment]
Ah right, we need to read the port from the config file now. Will be fixed with upcoming release.

**Comment by oradke** [comment]
Cool, thanks.

Von: MichaIng ***@***.***>
Gesendet: Sonntag, 16. Oktober 2022 21:58
An: MichaIng/DietPi ***@***.***>
Cc: oradke ***@***.***>; Author ***@***.***>
Betreff: Re: [MichaIng/DietPi] Cloudshell doesn't find PiHole anymore (Issue #5810)


Ah right, we need to read the port from the config file now. Will be fixed with upcoming release.

-

**Comment by oradke** [comment]
FYI, there no port listed in the config file by default! Might need to be activated first.

Am 16.10.2022 um 21:58 schrieb MichaIng ***@***.***>:

﻿

Ah right, we need to read the port from the config file now. Will be fixed with upcoming release.

**Comment by MichaIng** [comment]
Fixed with #5822 for release tomorrow.
